truffleruby/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
AFAIK it's no longer necessary to write `All rights reserved.`, and that also tends to be confusing (cc @wirthi).
Could you drop it here? (and in other files in this PR)

Reply:
This is the case I understand. Internal link: http://pdb.us.oracle.com/downloads/legal_notices.html